src/Apps/Consign/Routes/SubmissionFlow/Utils/validation.ts

```diff
@@ -47,6 +43,10 @@ export const artworkDetailsValidationSchema = yup.object().shape({
       coordinates: yup.array(yup.number()),
     })
     .required(),
+  category: yup
```

Comment:
This was likely an oversight - we weren't actually validating this, so skipping the 'Medium' drop-down still left you in a valid state to submit the form (repro'able in prod)!